docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
```

Comment:
It seems a little awkward to me when you are trying to list your own collections because it's not always easy to know your own account ID. It's also redundant since it is implied by your authentication. I think you could fix this by doing the following:

1. Remove `accounts` from the URL and just prefix with `collections`
2. Introduce a `user_id` filter parameter (for cases when you are selecting collections for other users)
3. Move these docs out of the `accounts` section and into a new `collections` section

Here is an example of what the new requests would look like:

Get your own collections:
````
GET /api/v3/collections/
````

If you have `Users:Edit` permissions, get other collections:
````
GET /api/v3/collections/?user_id=123
````

Get a specific collection:
````
GET /api/v3/collections/collection/321/
````